src/app/products/charges/create-charge/create-charge.component.scss

```diff
@@ -0,0 +1,15 @@
+.penalty-wrapper {
+  padding: 17.5px 0 17.5px 0;
+}
+
+.active-wrapper {
+  padding: 17.5px 0 17.5px 0;
+}
+
+.add-fee-frequency-wrapper {
+  padding: 17.5px 0 17.5px 0;
+}
+
+mat-divider {
+  position: relative !important;
```

Comment:
There is an angular material issue in which all mat-dividers within cards are set as position: absolute by default. This is necessary and there are no html classes or anything else to fix this.